drivers/platform/aducm3029/spi_extra.h

```diff
@@ -51,19 +51,6 @@
 /*************************** Types Declarations *******************************/
 /******************************************************************************/
 
-/**
- * @enum spi_channel
- * @brief Available SPI channels on the ADuCM3029
- */
-enum spi_channel {
```

Comment:
this change is not related to the commit description

Reply:
this is the same as device_id

Reply:
Change now realted to #998